### DISCUSSION TITLE
중급 프로젝트 1주차 회고

### DISCUSSION THREAD

**Discussion by siyeol97** [Party-Time-Job/Party-Time-Job#29] [회고록]
### keep
- 각자의 역할을 나누고 일을 진행함에 있어서 효율적이고 체계적인 것 같아서 좋습니다.
- 컨벤션과 규칙을 정하는데에 있어 막힘없이 정해져 수월하게 진행되는 느낌이 좋습니다.
- 새로운 라이브러리를 사용해서 경험할 수 있어서 좋습니다.

### problem
- 익숙한 라이브러리가 아닌 새로운 라이브러리를 학습하면서 진행해야해서 시간이 조금 오래 걸리지만 아직까지는 괜찮음.

### try
- 얼른 기본 요구사항을 끝내고 새로운 기능을 넣자!

**Comment by Crack-Egg** [comment]
## Keep

- 다들 정말 중급 프로젝트 시작 전부터 의지가 눈에 보일 정도로 열정적이여서 저도 같이 힘이 났던 것 같습니다. 이번 프로젝트에 있어 기본적인 규칙과 컨벤션 룰을 정하는데 수월하게 진행되었던 점과 의견 차이가 날 수 있었던 역할 분담에 있어서도 팀원들을 먼저 배려하는 모습들이 좋은 팀을 만났다고 느낄 수 있었습니다. 또한 모든 팀원들이 새로운 것에 도전하는데에 있어 정말 열정적인 것도 좋았던 것 같습니다.

## Problem

- 새로운 방식들을 학습하면서 진행하다보니 다소 진행하는데에 있어 속도면에 있어 느렸던 것 같습니다. 그 외에는 아직 초반이라 별 문제가 없는 것 같습니다.

## Try

- 가장 중요한 것은 부딪히면서 배우는 것이라고 생각합니다. 초반에 정했던 우선순위를 기준으로 빠르게 진행하고 부가적인 디자인과 기능들은 점차 완성하는 방향으로 진행하면 될 것 같습니다.

**Comment by DanteSnow** [comment]
## Keep

- 팀원들 모두가 다른 팀원을 먼저 배려해주려는 분위기가 정말 좋았습니다.
- 의견 대립이 생겼을 때에도 하나하나 귀 기울여주고, 반영하려고 노력하는 분위기가 정말 좋았습니다.
- 도전적이고 열정적이면서도 부드러운 팀을 만난 것 같아서 어떤 일이든 함께 해낼 수 있다는 확신이 들었습니다.

## Problem

- 제가 이사를 하게 되어서 팀 프로젝트에 온전히 에너지를 쏟지 못한다는 것이 유일한 문제인 것 같습니다.

## Try

- 팀 프로젝트를 위해서 이사를 빠르게 잘 진행해보겠습니다!
